crates/top/rerun/src/commands/entrypoint.rs

```diff
@@ -1216,16 +1225,20 @@ fn assert_receive_into_entity_db(
 
                 match msg.payload {
                     SmartMessagePayload::Msg(msg) => {
-                        let mut_db = match msg.store_id().kind() {
-                            re_log_types::StoreKind::Recording => rec.get_or_insert_with(|| {
-                                re_entity_db::EntityDb::new(msg.store_id().clone())
-                            }),
-                            re_log_types::StoreKind::Blueprint => bp.get_or_insert_with(|| {
-                                re_entity_db::EntityDb::new(msg.store_id().clone())
-                            }),
-                        };
-
-                        mut_db.add(&msg)?;
+                        if let DataSourceMessage::LogMsg(msg) = msg {
```

Comment:
an `if-let` without an `else` makes me nervous, especially in a test. I rather have a full match, with a comment explaining why some branches are ignored